docs/codeql/codeql-language-guides/using-api-graphs-in-python.rst

```diff
@@ -31,9 +31,9 @@ following snippet demonstrates.
 
 ➤ `See this in the query console on LGTM.com <https://lgtm.com/query/1876172022264324639/>`__.
 
-This query only selects the API graph node corresponding to the ``re`` module. To find
-where this module is referenced, you can use the ``getAUse`` method. The following query selects
-all references to the ``re`` module in the current database.
+This query selects the API graph node corresponding to the ``re`` module. This node represent the fact that the ``re`` module has been imported rather than a specific place in the program where the import happens. Therefore, there will be at most one result per project, and it will not have a useful location, so you have to click `Show 1 non-source result` in order to see it.
```

Comment:
Fixing the verb and suggesting two small improvements:
```suggestion
This query selects the API graph node corresponding to the ``re`` module. This node represents the fact that the ``re`` module has been imported rather than a specific location in the program where the import happens. Therefore, there will be at most one result per project, and it will not have a useful location, so you'll have to click `Show 1 non-source result` in order to see it.
```